packages/application-shell-connectors/src/components/application-context/application-context.spec.tsx

```diff
@@ -1,141 +1,172 @@
 import React from 'react';
-import { shallow, ShallowWrapper } from 'enzyme';
-import { ApplicationContextProvider } from './application-context';
+import { render, waitForElement, RenderResult } from '@testing-library/react';
```

Comment:
Why don't we use our own test-utils which exports this?

Reply:
It will introduce a cycle dependency, which I would like to avoid. Unless we extract the test-utils into a separate package...

Reply:
Hm. Okay. Maybe it then actually makes sense to have it on app-kits roadmap to have it as a package?

Reply:
Could be, but also it's only a "problem" with some of the app kit packages. In theory, using the test-utils is not necessary for most of them, since there is no need for an app-like setup.